added node binary as a config option

elmariachi111 opened the pull request:
I stumbled upon a problem when I integrated your project into my asset pipeline: 

```
[exception] 500 | Internal Server Error | Assetic\Exception\FilterException
[message] An error occurred while running:
'/Users/stefan/.npm-packages/lib/node_modules/node-sass/bin/node-sass' '--include-path' '/Users/stefan/work/web-fsv/apps/fsv/src/Fsv/ComparisonBundle/Resources/public/scss' '--output-style' 'nested' '--include-path' '/Users/stefan/work/web-fsv/apps/fsv/app/../vendor' '/private/var/tmp/assetic_sasssJfSBK' '--stdout'

Error Output:
env: node: No such file or directory
```

I added an option to configure the node executable as well and now it's running correctly: it's basically prepending /usr/local/bin/node to the cli. 

The option is backwards compatible: if one doesn't need it, he can omit it.

Commit:
added node binary as a config option

dmzkrsk commented:
Please remove unused `$rubyPath` from constructor's arguments and I'll merge it

Commit:
dropped ruby binary parameter (makes no sense in node-sass)

elmariachi111 commented:
I removed the ruby parameter and added the node/bin parameter to README. It's actually only needed when we're executing the filter from within an environment that doesn't contain node in its path. On Windows machines I'm fine not providing it (the node-sass binary is a shell script that invokes node) but on a Unix webserver it's invoked with a hashbang like `#!/usr/bin/env node` and the node might not be on the path. There are lots of people stumbling upon that issue from various contexts (google: "env: node: No such file or directory") and the default solution (symlink node in /usr/bin) just litters the global binary resolution.

dmzkrsk commented:
Merged
Version updated
Pushed to packagist
